src/main/java/seedu/smarthomebot/storage/StorageFile.java

```diff
@@ -5,12 +5,22 @@
 
 import java.util.logging.Logger;
 
+//@@author TanLeeWei
+
+/**
+ * Represent the parent class of ReadStorageFile and WriteStorageFile.
+ */
 public abstract class StorageFile {
 
     protected static ApplianceList applianceList;
     protected static LocationList locationList;
     public static Logger storageLogger = Logger.getLogger("SmartHomeBotLogger");
 
+    /**
+     * Passes the ApplianceList and LocationList to allow ReadStorageFile and WriteStorageFile to use.
+     * @param applianceList stores the appliances in SmartHomeBot
```

Comment:
@param applianceList List of  appliances in SmartHomeBot
@param locationList List of  locations in SmartHomeBot